libraries/abstractions/secure_sockets/utest/secure_sockets_utest.c

```diff
@@ -0,0 +1,1334 @@
+/*
+ * Amazon FreeRTOS HTTPS Client V1.1.1
```

Comment:
Is this version number in this header correct? Does not look like it?

Reply:
i recopied it from secure_sockets.c